socialhome/streams/app/stores/streamStore.js

```diff
@@ -1,31 +1,52 @@
 import Vue from "vue"
 import Vuex from "vuex"
-import isNil from "lodash/isNil"
+import ReconnectingWebSocket from "reconnecting-websocket"
 
-// Vue setup
+import getState from "streams/app/stores/streamStore.state"
+import {actions, mutations, stateOperations} from "streams/app/stores/streamStore.operations"
+
+/* This should actually rely in ``main.js``.
```

Comment:
Why should it be in `main.js`, seems like it should be here to me anyway since this is the store? Dunno :D Just thinking what is common in vue. Since everyone uses babel so they must be hitting the same thing and must already have a common way to deal with this.